learning/step_by_step/animations.rst

```diff
@@ -10,86 +10,74 @@ This tutorial will explain how everything is animated in Godot. Godot's
 animation system is extremely powerful and flexible.
 
 To begin, let's just use the scene from the previous tutorial (:ref:`doc_splash_screen`).
-The goal is, to add a simple animation to it. Here's a copy
+The goal is to add a simple animation to it. Here's a copy
 just in case: :download:`robisplash.zip <files/robisplash.zip>`.
 
-Creating the animation
-----------------------
+Add an animation player
+-----------------------
 
 First of all, add an :ref:`AnimationPlayer <class_AnimationPlayer>`
-node to the scene as a child of "bg" (the root node):
+node to the scene as a child of "background" (the root node):
```

Comment:
We should try and consistently follow the style guide conventions in all docs: http://docs.godotengine.org/en/latest/learning/scripting/gdscript/gdscript_styleguide.html

Which means the node should be named "Background" (capital "B").